### ISSUE TITLE
Selenium with headless Chrome not including 

### ISSUE THREAD

**Issue by darraghbuckley** [teamcapybara/capybara#2011]
## Meta
Capybara Version:
3.0.2
Driver Information (and browser if relevant):
selenium-webdriver

## Actual Behavior
``NameError: uninitialized constant Selenium
from /Users/darragh/.rvm/gems/ruby-2.4.2/gems/capybara-3.0.2/lib/capybara.rb:497:in `block in <top (required)>``

## Steps to reproduce
```
require 'capybara'

app = proc { |env| [200, { "Content-Type" => "text/html" }, [''] ] }

Capybara.server = :webrick
session = Capybara::Session.new(:selenium_chrome_headless, app)
session.visit("/")
```

The issue is specific to `selenium_chrome_headless`. The error looks to be because `selenium-webdriver` hasn't been `require`'d when it's referenced here:

https://github.com/teamcapybara/capybara/blob/master/lib/capybara.rb#L497

That gem is usually lazy-loaded when the driver is initialized here:

https://github.com/teamcapybara/capybara/blob/master/lib/capybara/selenium/driver.rb#L476

This looks to have been introduced in this change:

https://github.com/teamcapybara/capybara/commit/0275eab42c610cd1ccde7947b051d0b00857b9ce

The error looks to not be raised in tests because the tests independently require the gem here:

https://github.com/teamcapybara/capybara/blob/master/spec/selenium_spec_chrome.rb#L4

I can fix it on my end by requiring `selenium-webdriver` before touching Capybara. That's a satisfactory solution for me but it's kinda out of the pattern of the other drivers. If this is enough of an issue where you want a fix, I'm happy to put in a Pull Request. Otherwise, count this issue more as an FYI and feel free to close it.

**Comment by twalpole** [comment]
If you have time please submit a PR, otherwise I'll fix this sometime next week.

**Comment by twalpole** [comment]
Closed via 431482b751e5a65e9902eca1ece7abbd36646bee
